src/renderer/features/_shared/classes/DiceRoller.ts

```diff
@@ -191,25 +116,30 @@ class DiceRoller {
     }
 
     return {
-      total: total,
+      result: total,
       rolls: rolls,
     }
   }
 
-  public static _rollAccuracyDice(numberOfDice: number): object {
+  // parseRange = (text: string): { lower: number; upper: number; } => {
+  public static _rollAccuracyDice(numberOfDice: number): {result: number, rolls: number[] } {
```

Comment:
I think that if these are going to be public, we should drop the underscore prefix. Even if it's only ever externally called for testing, I think it's okay to have a totally valid and usable but currently-unused function out there than to bother with using the underscore on a public fn to denote something like "for testing only"

Reply:
Fair.  Changed for next push.